desktop/src/credits_scene_slc.nim

```diff
@@ -5,6 +5,15 @@ import scene_management
 from text_renderer import renderTextCached
 from game_input import wasClicked
 
+proc drawCreditSceneText(game: Game) =
```

Comment:
we don't really need a separate procedure for this, there isn't anything to really do here but render some text in the render proc.